Client/Ecosia/L10N/String.swift

```diff
@@ -92,5 +92,14 @@ extension String {
         case setAsDefaultBrowser = "Set Ecosia as default browser"
         case linksFromWebsites = "Links from websites, emails or messages will automatically open in Ecosia."
         case showTopSites = "Show Top Sites"
+        case everyTimeYouInvite = "Every time a friend accepts your invitation, you’ll contribute to plant extra trees."
+        case successfulInvites = "%d successful invite(s)"
```

Comment:
this could also go to the plurals stringsdict if QA complains about the plural "(s)"